rpcclient/wallet.go

```diff
@@ -1375,6 +1376,51 @@ func (c *Client) ListAccountsMinConf(minConfirms int) (map[string]dcrutil.Amount
 	return c.ListAccountsMinConfAsync(minConfirms).Receive()
 }
 
+// FutureGetMasterPubkeyResult is a future promise to deliver the result of a
+// GetMasterPubkeyAsync RPC invocation (or an applicable error).
+type FutureGetMasterPubkeyResult chan *response
+
+// Receive waits for the response promised by the future and returns a pointer to
+// the master extended public key for account and the network's hierarchical
+// deterministic extended key magic versions (e.g. MainNetParams)
+func (r FutureGetMasterPubkeyResult) Receive(net hdkeychain.NetworkParams) (*hdkeychain.ExtendedKey, error) {
+	res, err := receiveFuture(r)
+	if err != nil {
+		return nil, err
+	}
+
+	var pubkeystr string
+	err = json.Unmarshal(res, &pubkeystr)
+	if err != nil {
+		return nil, err
+	}
+
+	// pubkey is a hierarchical deterministic master public key (BIP0032)
```

Comment:
This is incorrect.  There is no guarantee what exact scheme it's using, but the current one is actually BIP0044, not 32.  I'd just remove the reference.

Reply:
Thank you. Are there any more changes that need to be made?

Reply:
Thanks.  Looks good.